pom.xml

```diff
@@ -629,6 +638,12 @@
                             </execution>
                         </executions>
                     </plugin>
+
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>sonar-maven-plugin</artifactId>
```

Comment:
I added the specific Maven SonarQube plugin configuration to the profile `test-coverage-nightly` as well, so it's the same as in `test-coverage`.